### ISSUE TITLE
Release for Lotus v1.28.0-rc-for-calibration-upgrade

### ISSUE THREAD

**Issue by BigLep** [filecoin-project/go-f3#383]
## Done Criteria
1. There is a tagged release that Lotus v1.28.0 can depend on that has the minimum F3 functionality needed for [Lotus v1.28.0 final calibration candidate](https://github.com/filecoin-project/lotus/issues/12109)
2. There is a PR to Lotus master depending on this version

## Items in scope
Generally, this corresponds with items in https://github.com/filecoin-project/go-f3/milestone/2.  

Specific items that must be in here that aren't completed yet at the time of this issue being created include:

Must haves:
- [ ] https://github.com/filecoin-project/go-f3/issues/388
- [x] https://github.com/filecoin-project/go-f3/issues/390
- [x] https://github.com/filecoin-project/go-f3/pull/348
- [x] enable F3 in calibration param 
- [x] #395 

Not blocking, but very nice to have - without these in the release, we will need to ask calib node operators to do another upgrade, for meaningful testing:
- [ ] https://github.com/filecoin-project/go-f3/issues/176
- [ ] https://github.com/filecoin-project/go-f3/issues/15

## Notes
I know the release is trivial, but I created this issue so: 
1. it could be linked from the [Lotus v1.28.0 release issue](https://github.com/filecoin-project/lotus/issues/12109).  I didn't want the Lotus issue to be tracking individual F3 items.  It should just be waiting for the go-f3 release.
2. make it clear that there is the open-a-depenency-bump step that needs to be done after the code is merged

**Comment by BigLep** [comment]
From @rjan90 : "2024-07-03: After discussions with F3 in the weekly sync - v0.0.3 is not a blocker for shipping RC2. I have therefore moved this down to Mainnet Upgrade Release"

As a result, I'm going to close this issue in favor of https://github.com/filecoin-project/go-f3/issues/386 .  We can list out there specific items that need to complete before mainnet.

**Comment by rjan90** [comment]
Reopening this ticket as https://github.com/filecoin-project/lotus/pull/12173 (and its dependency https://github.com/filecoin-project/go-f3/pull/391) has been surfaced to be Lotus v1.28.0-rc2 backport candidates, which will require a go-f3 release

**Comment by BigLep** [comment]
Per 2024-07-04 conversation with @rjan90, this can actually be closed now as Lotus 1.28.0-rc2 doesn't need another go-f3 release.  More context: https://filecoinproject.slack.com/archives/CP50PPW2X/p1720095087798859?thread_ts=1720084473.124679&cid=CP50PPW2X
